Demo/Src/main.c

```diff
@@ -269,4 +273,34 @@ void assert_failed(uint8_t *file, uint32_t line)
 }
 #endif  /* USE_FULL_ASSERT */
 
+/**
+    Wire up the `stdio` system call, so that `printf()`
+    works as a logging message generator.
+
+    @param  file    The log entry -- a C string -- to send.
+    @param  ptr     A pointer to the C string we want to send.
+    @param  length  The length of the message.
+
+    @return         The number of bytes written, or -1 to indicate error.
+ */
+int _write(int file, char *ptr, int length) {
+    if (file != STDOUT_FILENO) {
+        errno = EBADF;
+        return -1;
+    }
+
+    // Write out the message string. Each time confirm that Microvisor
+    // has accepted the request to write data to the channel.

```

Comment:
nit: commentary talks about old concepts ('channel').

Reply:
Fixed up the language in this one.